setup.py

```diff
@@ -32,6 +32,7 @@
         "sphinxcontrib-redoc>=1.5.1",
     ],
     "tests": tests_require,
+    "ssh": ["paramiko[gssapi]>=2.6.0"]
```

Comment:
If `paramiko` *et al* needs to be moved out of installation requirements for ReadTheDocs to work, then we need to create a new file `requirements.in` where it shoulld be added, so that `pip-compile` would generate proper `requirements.txt` out of all our dependencies. You can test via `pip-compile --output-file=requirements.txt` and see what it includes

Reply:
Adding it.